clients/banking/src/components/AccountArea.tsx

```diff
@@ -805,6 +805,9 @@ export const AccountArea = ({ accountMembershipId }: Props) => {
                                         params={params}
                                         currentUserAccountMembership={currentUserAccountMembership}
                                         canOrderPhysicalCards={canOrderPhysicalCards}
+                                        b2BMembershipIDVerification={
```

Comment:
let's rename this prop `shouldDisplayIdVerification={projectInfo.B2BMembershipIDVerification !== false}` like it's done elsewhere

Reply:
thanks a lot I didn't see it. Changed in [bad3092](https://github.com/swan-io/swan-partner-frontend/pull/189/commits/bad3092e9aaf7a58e95d3cbeb72d943d433833f8)